java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -372,6 +386,157 @@ public void computeResult_resultAlwaysInsideProvidedBoundaries() {
     }
   }
 
+  /** Calling computeConfidenceInterval before noised mean is computed throws an exception. */
+  @Test
+  public void computeConfidenceInterval_calledBeforeResult() {
+    assertThrows(IllegalStateException.class, () -> mean.computeConfidenceInterval(ALPHA));
+  }
+
+  @Test
+  public void computeConfidenceInterval_callsNoiseComputeConfidenceIntervalCorrectly() {
+    int maxPartitionsContributed = 1;
+    int maxContributionsPerPartition = 3;
+    double alpha = 0.5;
+    mean =
```

Comment:
can't we use default mean for that test? the one which was created during the set up?

Reply:
https://github.com/google/differential-privacy/blob/b3e72b7593703fbc77fc29bdb2c6b7b73e5067fc/java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java#L193

Even though it is equal to default mean it is still there in the code in computeResult_callsNoiseCorrectly().
Also if I am testing it is passing parameters correctly, isn't better to clarify in the same block what parameters I am using?

Reply:
I don't think this example shows that we just want to list all used params in the tests, they used new one because new parameters are interesting in terms that sensitivities were calculated and passed correctly, etc.

If in your test parameters which you used are also "more interesting" that in the default version then you can use it as well. If not (which means with default parameters we can show as well that sensitivities, etc were calculated correclty) then better to use  the default one.

Reply:
As we discussed offline, We also agreed to keep it with non-default values (e.g non-equal maxPartitionsContributed, maxContributionsPerPartition since they are equal in the default mean) to also detect that sensitivities were calculated and passed correctly which proved to be useful in #53